docs/self-hosting/deployment-options/standalone-infisical.mdx

```diff
@@ -26,9 +26,9 @@ Prerequisites:
 
     ```bash
     docker run -p 80:8080  \
-    -e ENCRYPTION_KEY=f40c9178624764ad85a6830b37ce239a \
     -e AUTH_SECRET="q6LRi7c717a3DQ8JUxlWYkZpMhG4+RHLoFUVt3Bvo2U=" \
-    -e MONGO_URL="<>" \
+    -e DB_CONNECTION_URI="<>" \
+    -e REDIS_URL="<>" \
     infisical/infisical:latest
```

Comment:
This is the image for mongo. The new image `latest` is under `infisical/infisical:latest-postgres`